types/src/transaction/metadata.rs

```diff
@@ -25,23 +34,30 @@ pub struct GeneralMetadataV0 {
     // Subaddress from which the funds are being sent
     #[serde(with = "serde_bytes")]
     from_subaddress: Option<Vec<u8>>,
-    // Event sequence number of referenced payment
+    // In the case of refunds, referenced_event refers to the event sequence
+    // number of the sender’s original sent payment event.
+    // Since refunds are just another form of P2P transfer, the referenced
+    // event field allows a refunded payment to refer back to the original
+    // payment
     referenced_event: Option<u64>,
 }
 
-// Used for versioning of travel rule metadata
+/// List of supported transaction metadata format versions for transactions
+/// subject to travel rule
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
 pub enum TravelRuleMetadata {
     TravelRuleMetadataVersion0(TravelRuleMetadataV0),
 }
 
+/// Transaction metadata format for transactions subject to travel rule
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
 pub struct TravelRuleMetadataV0 {
     // Off-chain reference_id.  Used when off-chain APIs are used.
```

Comment:
I suspect you need `///` here too for this to show up in docs.

Reply:
Fixed